include/admin.jsonnet

```diff
@@ -85,7 +85,16 @@ local common = import "common.jsonnet";
                       {
                         "name": "CONFIGURATION_SERVICE_PORT",
                         "value": "5672"
-                      }]),
+                      },
+                      {
+                        "name": "API_SERVER_HOSTNAME",
```

Comment:
So my reason for using this name is that the API  would eventually be part of the enmasse API server which would be a 3rd party API server in kubernetes. So the name is following the kubernetes convention in that sense.